js/repeater.js

```diff
@@ -698,6 +698,21 @@
 			var viewTypeObj = {};
 			var height;
 			var viewportMargins;
+			var $hiddenParents = [];
+			var previousProperties = [];
+			var $previousParents = this.$element.parentsUntil(':visible').addBack();
+			var currentParent;
```

Comment:
looking at the repeater element as well so this variable name could be `$currentHiddenElement`